tensorflow/tools/docker/Dockerfile.devel-mkl-horovod

```diff
@@ -29,7 +29,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         openjdk-8-jdk \
         openjdk-8-jre-headless \
         wget \
-        numactl \
+        libnuma-dev \
```

Comment:
FYI, we cannot distribute numa ourselves, but I think just having a dockerfile with reference to it is fine.
@martinwicke to confirm.

Reply:
Yes, it's fine.

Reply:
Yes, we are not distributing binary, this is just a reference in Dockerfile. Thanks!